src/utils/globals.ts

```diff
@@ -93,6 +93,19 @@ export const highestCharacterStats = (
   };
 };
 
+export const doStatsChange = (characterObject: CharacterObject): boolean => {
+  const { phases } = characterObject;
+  const activePhase = phases[phases.length - 1];
+
+  const startingKeyFrame = activePhase.attributesKeyFrames[0];
+  const finalKeyFrame =
+    activePhase.attributesKeyFrames[activePhase.attributesKeyFrames.length - 1];
+
+  return (
+    JSON.stringify(startingKeyFrame.data) !== JSON.stringify(finalKeyFrame.data)
```

Comment:
This works as long as the properties are ordered in the same way, yes? I think it should work correctly, but hopefully HG never changes those...

Reply:
yeah, couldn't think of a better way to compare equality

if HG changed the order of their own keys, within the same attribute keyframe of ONE summon, I would be highly confused

Reply:
I mean, it's valid in this case for the assumptions we've made, which feel reasonable enough.

We could bring in a `isEqual` dependency like `lodash.isequal`, but not much sense in doing so unless we need it.